tools/install.py

```diff
@@ -38,7 +38,7 @@ def run(target, local, user, config, toolchain, **kwargs):
     if not local:
         num_robots = 4
         target = {
-            "{}{}".format(k, num): "10.1.1.{}".format(num)
+            "{}{}".format(k, num): "10.1.0.{}".format(num)
```

Comment:
How come we changing this - is there an actual reason to change the ip from `10.1.1.x` to `10.1.0.x`?

Reply:
When I copied the `ip` from the robot, that's what it was. I think it may have been accidentally changed :shrug:

Reply:
Those IPs are correct so long as the network is still how I set it up ages ago.

10.1.0.X is for robots (where X is the robot number), I think NUgus 1,2,3 etc ended up with 0.{11,12,13} since 0.{1,2,3} were the darwins
10.1.1.X is for non robots via DHCP for normal network stuff
Mask should be 255.255.0.0 (or maybe 255.255.254.0)

Reply:
They've been 10.1.1.x {1,2,3,...} for the NUgus' for a few years. Non-robots seem to connect as 10.1.0.x as far as I've seen.

Reply:
Perhaps I got it backwards :P but something like that

Reply:
I think they were changed at, or perhaps before robocup for some reason. @Tom0Brien might know

Reply:
I think maybe at RoboCup we were just following @TrentHouliston's instructions a little too closely :laughing: 
we should put them all back to 10.1.1.x